orchagent/fdborch.cpp

```diff
@@ -1418,6 +1418,17 @@ bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name,
             {
                 SWSS_LOG_ERROR("macUpdate-Failed for attr.id=0x%x for FDB %s in %s on %s, rv:%d",
                             itr.id, entry.mac.to_string().c_str(), vlan.m_alias.c_str(), port_name.c_str(), status);
+                if ((SAI_STATUS_ITEM_NOT_FOUND  == status) && (fdbData.origin == FDB_ORIGIN_MCLAG_ADVERTIZED)) {
```

Comment:
Can you add comments here as you have explained in the PR for the Add case here as well?  Reason is by the time this PR is merged, reader will have a hard time to understand why this code was like this.
when I review this part, I had to go back to your PR description again in order to understand why...
Please add appropriate comment.

Reply:
Thanks @gechiang . I have added comments now.